lib/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+set(KICKCAT_SOURCES 
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/Error.cc
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/protocol.cc
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/Time.cc
+)
+
+add_library(kickcat ${KICKCAT_SOURCES})
+target_include_directories(kickcat PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include)
+target_include_directories(kickcat PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/include/kickcat) # Does not work with private
```

Comment:
I do not understand the comment here?

Reply:
The problem has been fixed, I forgot to remove the comment.

Reply:
BTW I think you can do only one call with PUBLIC and PRIVATE in a row

Reply:
yes it works